packages/transports/aws-cloudwatch-logs/src/vendor/exit-hook.ts

```diff
@@ -0,0 +1,116 @@
+// Stripped version of: https://github.com/sindresorhus/exit-hook/blob/main/index.js
+
```

Comment:
Also note what the differences are between the original and your modification in a comment or a readme in this dir for the event you may need to revisit this code in the future and might need to pull in changes from the original project